src/renderer/RenderContext.cpp

```diff
@@ -1,16 +1,24 @@
 #include "renderer/RenderContext.hpp"
-
+#include <iostream>
 void RenderContext::Init(const char *title, int x, int y, int w, int h)
 {
+    if (SDL_Init (SDL_INIT_EVENTS | SDL_INIT_VIDEO) < 0)
```

Comment:
SDL is initialized in main.cpp, why is this here again?

Reply:
...